ui/src/providers/SavedSets.js

```diff
@@ -36,7 +36,7 @@ const SavedSetsProvider = (props) => {
             data: {
               saveSet: { setId, ids },
             },
-          } = await apiFetcher({ query: fetchSetsQuery, sqon });
+          } = await apiFetcher({ query: fetchSetsQuery, variables: { sqon } });
```

Comment:
SavedSets further updated in theming branch: https://github.com/nci-hcmi-catalog/portal/pull/1107/files#diff-348f7bccd957afe8afd812935e054508f93b322b852aecbf0e231db0bee3f023